tools/loader/ModelRunner.cpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Loader.h"
+
+#include "glow/Graph/Nodes.h"
+#include "glow/Importer/Caffe2.h"
+#include "glow/Importer/ONNX.h"
+
+#include "llvm/Support/raw_ostream.h"
+
+using namespace glow;
+
+int main(int argc, char **argv) {
+  // The loader verifies/initializes command line parameters, and initializes
+  // the ExecutionEngine and Function.
+  Loader loader(argc, argv);
+
+  // Create the model based on the input net, and get SaveNode for the output.
+  SaveNode *output;
+
+  ProtobufLoader *LD;
+  if (!loader.getCaffe2NetDescFilename().empty()) {
+    LD = new caffe2ModelLoader(loader.getCaffe2NetDescFilename(),
+                               loader.getCaffe2NetWeightFilename(), {}, {},
+                               *loader.getFunction());
+  } else {
+    LD = new ONNXModelLoader(loader.getOnnxModelFilename(), {}, {},
+                             *loader.getFunction());
+  }
+  output = LD->getRoot();
+
+  // Compile the model, and perform quantization/emit a bundle/dump debug info
+  // if requested from command line.
+  loader.compile();
+
+  // If in bundle mode, do not run inference.
+  if (!emittingBundle()) {
```

Comment:
Out-of-curiosity, what are the externally observable effect if emit-bundle is not set?
Basically, I am wondering if we want this binary to do something externally visible to tell the user that it did something. Like printing loading and compilation succeeded.

Reply:
If emit-bundle is not set, then the branch gets executed and obviously something gets printed.

If emit-bundle is set, then unfortunately there's no output printed to stdout. We may want to fix it in follow up PR. Note, that the fix will be in `Loader.cpp` file, which is shared between ModelRunner and ImageClassifier.